ivy.xml

```diff
@@ -18,6 +18,7 @@
   <dependencies defaultconf="default->default">
 
     <dependency org="commons-codec"       name="commons-codec"       rev="1.5"              transitive="false"/>
+    <dependency org="com.google.guava"    name="guava"               rev="17.0"             transitive="false"/>
```

Comment:
This should probably use a variable for revision, specifically the variable and value used in Kettle.